.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -38,6 +38,8 @@ jobs:
       KATA_HYPERVISOR: ${{ matrix.vmm }}
       KUBERNETES: "k3s"
       USING_NFD: "true"
+      SNAPSHOTTER: ${{ matrix.snapshotter }}
```

Comment:
It is missing the `snapshotter` entry on `run-kata-deploy-tests-on-tdx` like you did on the other jobs.

Reply:
Fixed.